docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -62,12 +62,12 @@ views for displaying drilldown pages for date-based data.
       month or day using the attribute ``date_list_period``. This also applies
       to all subclass views.
 
-    **Example myapp/urls.py**::
+    **Example urls.py**::
```

Comment:
I am tempted to either leave these alone or turn these into captions :thinking: 
If they get turned into captions this might be a new commit and so I'd wait to hear what other people think before making an update

Reply:
ok

Reply:
It seems okay to me to remove the `myapp/` prefix since class-based-views/base.txt doesn't include it.